starter-code/public/scripts/controllers/routes.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 var app = app || {};
 
-// TODO: Configure routes for this app with page.js, by registering each URL your app can handle, linked to a a single controller function to handle it. Note that these routes do not need to wrapped in an IIFE.
-
-// TODO: What function do you call to activate page.js? Fire it off now, to execute. Note that it does not need to be attached to the 'app' object nor wrapped in an IIFE.
+// TO/DO: Configure routes for this app with page.js, by registering each URL your app can handle, linked to a a single controller function to handle it. Note that these routes do not need to wrapped in an IIFE.
+page('/articles', app.articleController.init);
```

Comment:
This should be `/` to ensure the list of articles is shown from the home page. Without a route for `/`, clicking Home in the lab produces a full page request instead of staying in the single-page app.

`/articles` is the server-side route that returns the list of articles as JSON. It does not need a client-side route counterpart.